packages/sanity/src/core/store/_legacy/document/document-pair/utils/isLiveEditEnabled.ts

```diff
@@ -1,5 +1,4 @@
 import {type Schema} from '@sanity/types'
 
 export const isLiveEditEnabled = (schema: Schema, typeName: string): boolean =>
-  (globalThis as any).__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED_LIVE_EDIT_OVERRIDE ??
```

Comment:
Curious why you are removing this
It's a scary env!

Reply:
It was initially added as part of my hacky (and temporary) [implementation](https://github.com/sanity-io/sanity/commit/c74da9ce626e42b407b7529993fc6b8292151963) of version viewing and editing.

This branch reverts that commit 🙂. If this scary env goes to production, I've messed up!

Reply:
Sounds good! haha